standalone/src/test/java/io/pravega/local/TlsProtocolVersion12Test.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.local;
+
+        import org.junit.ClassRule;
+        import org.junit.Test;
+
+        import static io.pravega.local.PravegaSanityTests.testWriteAndReadAnEvent;
+
+public class TlsProtocolVersion12Test {
+
+    @ClassRule
+    public static final PravegaEmulatorResource EMULATOR = new PravegaEmulatorResource.PravegaEmulatorResourceBuilder(false, true, false).tlsProtocolVersion("TLSv1.2").build();
```

Comment:
Why is this static? It will essentially be shared with all other tests which may prevent you from running tests in parallel. Is this intentional?

Reply:
Yes, this is static because Fields and Methods with @ClassRule annotation must be public and static.